lib/spree/shipment_notice.rb

```diff
@@ -60,7 +60,6 @@ def ship_it!
       shipment.update_attribute(:tracking, tracking)
 
       unless shipment.shipped?
-        shipment.reload.ready! if shipment.pending?
```

Comment:
Yeah, in Payment there is a callback, that updates the associated order and this callback uses
OrderUpdater to update the shipments too.

```ruby
   # Spree::Payment
   # update the order totals, etc.
    after_save :update_order
```